Sources/AsyncHTTPClient/HTTPHandler.swift

```diff
@@ -162,12 +162,14 @@ extension HTTPClient {
                 }
             }
 
-            func supports(scheme: String) -> Bool {
+            func supportsRedirects(to scheme: String?) -> Bool {
```

Comment:
Why has this become optional?

Reply:
Mostly to simplify the call site. I can change that to

```swift
guard let scheme = url.scheme, self.request.kind.supportsRedirects(to: scheme) {...
```

if you’d prefer.

Reply:
Nah, this works too, just wanted to be sure I understood it.